spec/factories/tasks.rb

```diff
@@ -5,5 +5,11 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+    project_id { 1 }
```

Comment:
idを指定しているのは`belongs_to`のオプションによる必須項目のためでしょうか？
そうであれば、idの指定ではなく関連するデータを作成する形にしましょう！
```suggestion
    association :project
```

Reply:
07b8f8f にて修正しました。